packages/modules/widget/src/errors.ts

```diff
@@ -1,18 +1,22 @@
 type WidgetErrorType = 'not_found' | 'unauthorized' | 'unknown';
 
-export class GetWidgetError extends Error {
+export class GetWidgetManifestError extends Error {
```

Comment:
Could you give this error a better name :)
example WidgetManifestLoadError

> ps, in the constructor, give the error a name, so it can be checked between instances